flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/io/StreamOneInputProcessor.java

```diff
@@ -62,6 +62,16 @@ public StreamOneInputProcessor(
 		return input.getAvailableFuture();
 	}
 
+	@Override
+	public boolean isAvailable() {
+		return input.isAvailable();
+	}
+
+	@Override
+	public boolean isApproximatelyAvailable() {
+		return input.isApproximatelyAvailable();
+	}
+
```

Comment:
Why do we need this commit? The default implementations should be doing the same thing.

Reply:
It is the same if the input does not override default implementations.

Mine does.